src/client/driver/command-executors/execute-action.js

```diff
@@ -202,7 +203,14 @@ export default function executeAction (command, elementAvailabilityTimeout) {
                 ]);
             })
             .then(() => resolve(new DriverStatus({ isCommandResult: true })))
-            .catch(err => resolve(new DriverStatus({ isCommandResult: true, executionError: err })));
+            .catch(err => {
+                // NOTE: in case if we couldn't find element for events
+                // simulation we raise this type of error (GH - 337)
```

Comment:
```
                // NOTE: in case we couldn't find an element for event
                // simulation, we raise an error of this type (GH - 337)
```